### PR TITLE
[NETBEANS-1047] Don't add whitespace to a wrong place when textual operators are used

### PR DESCRIPTION
**Pull request by junichi11**
https://issues.apache.org/jira/browse/NETBEANS-1047

Related to https://netbeans.org/bugzilla/show_bug.cgi?id=240274

### CODE REVIEW

**Comment by Baneeishaque** [comment]
Is the issue exist?

I used php in 8.2 version for my production works, i didn't encounte this issue.

K, i will try these works on nb 11.

**Comment by junichi11** [comment]
Yes, we can reproduce it in NetBeans 8.2.
netbeans1047

**Comment by Baneeishaque** [comment]
Let me check.

File: php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java [hunk 1]
```diff
@@ -2558,4 +2560,9 @@ protected static boolean isWhitespace(final CharSequence text) {
     private static boolean isAnonymousClass(ASTNode astNode) {
         return astNode instanceof ClassInstanceCreation && ((ClassInstanceCreation) astNode).isAnonymous();
     }
+
+    private static boolean isPHPOperator(PHPTokenId id) {
```
Comment (tmysik) [inline php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java]: Could be maybe moved to some `util` class as it seems to me to be quite useful in general. Not sure about that so just a hint.

**Comment by tmysik** [comment]
Please, let me know about my hint above, I am ready to merge.

Thanks!

File: php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java (hunk 1 above)
Comment (junichi11) [reply php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java]: Will do it. Thanks!

**Comment by junichi11** [comment]
@tmysik Fixed. Thank you!

**Pull request merged**